src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -4286,6 +4324,12 @@ components:
           example: server123
           minLength: 1
           maxLength: 255
+        server_address:
+          type: string
+          format: text
+          description: security server ip address
```

Comment:
maybe description should say just "address" or something, since it is not guaranteed to be an _ip address_, can be a name also

Reply:
Improved the description